<launch>

    <arg name="debug" default="false" />

    <node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen" required="true" ns="autonomous" unless="$(arg debug)">
        <param name="width" type="int" value="640" />
        <param name="height" type="int" value="480" />
        <param name="fps" type="int" value="25" />
        <param name="frame" type="string" value="wide_stereo" />
        <param name="format" type="string" value="yuyv" />
        <param name="auto_focus" type="bool" value="False" />
        <param name="focus_absolute" type="int" value="0" />
        <!-- other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->

        <param name="device" type="string" value="/dev/video1" />
    </node>

</launch>
